exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillConnectionImpl.java

```diff
@@ -369,24 +370,33 @@ AvaticaFactory getFactory() {
     return factory;
   }
 
+  private static void closeOrWarn(final AutoCloseable autoCloseable, final String message, final Logger logger) {
```

Comment:
should we make this public and move this to `org.apache.drill.common.AutoCloseables` ?